### ISSUE TITLE
leaderboard tri-M ICCV21

### ISSUE THREAD

**Issue by csyanbin** [google-research/meta-dataset#75]
@lamblin @eleniTriantafillou @vdumoulin  Can you help to add these results to the leaderboard? Thanks! 

### ***tri*-M**

```
triM_all_df = pd.DataFrame(
    columns=['mean (%)', '95% CI', '# episodes'],
    index=datasets
)
triM_all_df['# episodes'] = 600
triM_all_df.loc[1:, ['mean (%)', '95% CI']] = [
    [58.6, 1.0],
    [92.0, 0.6],
    [82.8, 0.7],
    [75.3, 0.8],
    [71.2, 0.8],
    [77.3, 0.7],
    [48.5, 1.0],
    [90.5, 0.5],
    [63.0, 1.0],
    [52.8, 1.1]
]
triM_all_df

ref = ("Liu et al. (2021)",
       "Yanbin Liu, Juho Lee, Linchao Zhu, Ling Chen, Humphrey Shi, Yi Yang; "
       "[_A Multi-Mode Modulator for Multi-Domain Few-Shot Classification_]"
       "(https://openaccess.thecvf.com/content/ICCV2021/papers/Liu_A_Multi-Mode_Modulator_for_Multi-Domain_Few-Shot_Classification_ICCV_2021_paper.pdf); "
       "To appear in ICCV 2021.")
references.append(ref)
```

These results have the shuffle buffer issue fixed for Traffic Sign, and posted the updated results on our supplementary: https://openaccess.thecvf.com/content/ICCV2021/supplemental/Liu_A_Multi-Mode_Modulator_ICCV_2021_supplemental.pdf

**Comment by eleniTriantafillou** [comment]
done, thanks for submitting your results to the leaderboard!

**Comment by vdumoulin** [comment]
Fixed via 142ea56.
